.changeset/nine-chicken-fly.md

```diff
@@ -0,0 +1,13 @@
+---
+"@mysten/wallet-adapter-wallet-standard": patch
+"@mysten/wallet-adapter-unsafe-burner": patch
+"@mysten/wallet-adapter-base": patch
+"@mysten/wallet-adapter-all-wallets": patch
+"@mysten/wallet-kit-core": patch
+"@mysten/wallet-standard": patch
+"@mysten/wallet-kit": patch
+"@mysten/sui.js": patch
+"@mysten/bcs": patch
```

Comment:
This seems to be off. Can you delete this file and re-run `pnpm changeset` and select `minor` for `"@mysten/sui.js"`? It should look like something like this: https://github.com/MystenLabs/sui/pull/8551/files#diff-eb722c7552ef8ef7d07f811137f19585201f3b824a548a927111700b618cc8de

Reply:
Hmm this is what I get when I run `pnpm changeset`, which seems to say that I changed all of the packages? If not how can I tell which packages I changed?
```
🦋  Which packages would you like to include? … 
◯ changed packages
  ◯ @mysten/wallet-adapter-wallet-standard
  ◯ @mysten/wallet-adapter-unsafe-burner
  ◯ @mysten/wallet-adapter-base
  ◯ @mysten/wallet-adapter-all-wallets
  ◯ @mysten/wallet-kit-core
  ◯ @mysten/wallet-standard
  ◯ @mysten/wallet-kit
  ◯ @mysten/sui.js
  ◯ @mysten/bcs
```